packages/grid/_modules_/grid/models/colDef/gridStringColDef.ts

```diff
@@ -5,6 +5,7 @@ import { getGridStringOperators } from './gridStringOperators';
 
 export const GRID_STRING_COL_DEF: GridColTypeDef = {
   width: 100,
+  minWidth: 10,
```

Comment:
Is not 10px too little? Now if I resize a column, the separator invades the 3 dot icon.

<img width="150" src="https://user-images.githubusercontent.com/42154031/126198724-82cd961c-9ab7-40e3-8182-cd6fa99a7930.png" alt="image" style="max-width:100%;">

```suggestion
  minWidth: 48,
```